packages/bundlers/default/src/DefaultBundler.js

```diff
@@ -290,10 +306,19 @@ function deduplicateBundle(bundleGraph: MutableBundleGraph, bundle: Bundle) {
     for (let asset of assets) {
       if (
         bundle.hasAsset(asset) &&
-        bundleGraph.isAssetInAncestorBundles(bundle, asset)
+        (bundleGraph.isAssetInAncestorBundles(bundle, asset) ||
+          isAssetInSiblingBundleGroups(bundleGraph, bundle, asset))
```

Comment:
I noticed this as well. Should we expose `isAssetInSublingBundleGroups` on the bundleGraph?

Reply:
Is it also possible for siblings of ancestor bundles to include the asset? Maybe we should change `isAssetInAncestorBundles` to check siblings of ancestors as well, and change its name to reflect that.

Reply:
oooh maybe. I hadn't considered that... 🤔